### PR TITLE
implemented test for privacy Policy Page.

### PR DESCRIPTION
**Pull request by MuyanjaKevin**
Made tests for privacy policy page.

### CODE REVIEW

**Comment by marcinkunert** [comment]
Closes #12

File: src/test/java/net/eventsexpress/app/tests/PrivacyPageTest.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    @Test
+    public void testThatUserIsRedirectedToPrivacyPolicyPage() {
+        footer.scrollToFooter();
+        footer.click_privacy_link();
+        PrivacyPage.assertPrivacyHeadingIsDisplayed();
+
+
+
+
+    }
```
Comment (marcinkunert) [inline src/test/java/net/eventsexpress/app/tests/PrivacyPageTest.java]:
```java
public class PrivacyPageTest extends BaseTest {

    @Test
    public void testThatUserIsRedirectedToPrivacyPolicyPage() {
        new Footer()
                .scrollToFooter()
                .clickPrivacyLink();

        PrivacyPage privacyPage = new PrivacyPage();
        privacyPage.assertPrivacyHeadingIsDisplayed();
    }

}
```

File: src/test/java/net/eventsexpress/app/pages/Footer.java [hunk 2]
```diff
@@ -16,4 +16,7 @@ public void scrollToFooter() {
         Base.scrollToElementXpath(FOOTER_XPATH);
     }
 
+    public void click_privacy_link(){Base.findElementByXpath(PRIVACY_LINK_XPATH).click();}
```
Comment (marcinkunert) [inline src/test/java/net/eventsexpress/app/pages/Footer.java]: code formatting

**Review by marcinkunert — CHANGES_REQUESTED** [review]

File: src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java [hunk 3]
```diff
@@ -19,5 +19,9 @@ public PrivacyPage() {
     public String getPrivacyHeading() {
         return privacyHeading.getText();
     }
+    public static void assertPrivacyHeadingIsDisplayed() {
+        assert Base.findElementByCSS(PRIVACY_HEADING_CSS).isDisplayed()
```
Comment (servettonga) [inline src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java]: Use the @FindBy annotation, Base page is deprecated.

**Review by servettonga — CHANGES_REQUESTED** [review]

File: src/test/java/net/eventsexpress/app/tests/PrivacyPageTest.java (hunk 1 above)
Comment (MuyanjaKevin) [reply src/test/java/net/eventsexpress/app/tests/PrivacyPageTest.java]: I have made necessary changes according to the review

File: src/test/java/net/eventsexpress/app/pages/Footer.java (hunk 2 above)
Comment (MuyanjaKevin) [reply src/test/java/net/eventsexpress/app/pages/Footer.java]: correctly formatted the code as requested.

File: src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java (hunk 3 above)
Comment (MuyanjaKevin) [reply src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java]: Made new method that uses @findby annotation as requested

File: src/test/java/net/eventsexpress/app/pages/Footer.java [hunk 4]
```diff
@@ -28,4 +28,6 @@ public Footer navigateToPrivacyPage() {
         return this;
     }
 
+
```
Comment (servettonga) [inline src/test/java/net/eventsexpress/app/pages/Footer.java]: remove this extra line

File: src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java [hunk 5]
```diff
@@ -1,12 +1,20 @@
 package net.eventsexpress.app.pages.footerpages;
 
+import net.eventsexpress.app.config.ConfigurationManager;
```
Comment (servettonga) [inline src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java:4]: Order imports by their source

File: src/test/java/net/eventsexpress/app/tests/PrivacyPageTest.java [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+}
+
+
+
+
+
+
+
+
```
Comment (servettonga) [inline src/test/java/net/eventsexpress/app/tests/PrivacyPageTest.java]: remove extra lines, leave only one at the bottom

Comment (MuyanjaKevin) [reply src/test/java/net/eventsexpress/app/tests/PrivacyPageTest.java]: extra have been deleted

File: src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java (hunk 5 above)
Comment (MuyanjaKevin) [reply src/test/java/net/eventsexpress/app/pages/footerpages/PrivacyPage.java:4]: i have ordered the imports accordingly

File: src/test/java/net/eventsexpress/app/pages/Footer.java (hunk 4 above)
Comment (MuyanjaKevin) [reply src/test/java/net/eventsexpress/app/pages/Footer.java]: i have removed extra line

**Review by marcinkunert — APPROVED** [review]

**Review by servettonga — APPROVED** [review]

**Pull request merged**